src/main/views/courts/tabs/phoneNumbersContent.njk

```diff
@@ -35,19 +35,27 @@
 
 {% for contact in contacts %}
 
+  {% set isNew = (contact.isNew === true) or (contact.isNew === "true") %}
```

Comment:
Why can isNew be both a boolean and a string in terms of the condition? :D

Reply:
isNew is a hidden input on the form. It will always be posted back as a string, despite the type being boolean. This is just to ensure either the boolean or string form will work.